mappings/net/minecraft/world/poi/PointOfInterestStorage.mapping

```diff
@@ -3,24 +3,37 @@ CLASS net/minecraft/unmapped/C_uegwgivt net/minecraft/world/poi/PointOfInterestS
 	FIELD f_bwcnjgqz preloadedChunks Lit/unimi/dsi/fastutil/longs/LongSet;
 	FIELD f_kjhqnjbr MAX_VILLAGE_DISTANCE I
 	FIELD f_zaepnyld VILLAGE_SECTION_SIZE I
+	METHOD <init> (Lnet/minecraft/unmapped/C_tzvgbkzf;Ljava/nio/file/Path;Lcom/mojang/datafixers/DataFixer;ZLnet/minecraft/unmapped/C_wqxmvzdq;Lnet/minecraft/unmapped/C_qpninoyb;)V
+		ARG 1 storageInfo
+		ARG 2 directory
+		ARG 3 dataFixer
+		ARG 4 dsync
+		ARG 5 registryManager
+		ARG 6 world
 	METHOD m_afhakdbq (Lnet/minecraft/unmapped/C_aurosfgf;Lnet/minecraft/unmapped/C_zubvmeye;Lnet/minecraft/unmapped/C_hdurmsnk;)V
 		ARG 3 set
 	METHOD m_agkornyv getType (Lnet/minecraft/unmapped/C_hynzadkk;)Ljava/util/Optional;
 		ARG 1 pos
+	METHOD m_bnugafgz getNearestPos (Ljava/util/function/Predicate;Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_hynzadkk;ILnet/minecraft/unmapped/C_uegwgivt$C_dzezdmdw;)Ljava/util/Optional;
+		ARG 2 posPredicate
 	METHOD m_dsxxnxyk getInCircle (Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_hynzadkk;ILnet/minecraft/unmapped/C_uegwgivt$C_dzezdmdw;)Ljava/util/stream/Stream;
 		ARG 1 typePredicate
 		ARG 2 pos
 		ARG 3 radius
 		ARG 4 occupationStatus
 	METHOD m_evxnnkpe getDistanceFromNearestOccupied (Lnet/minecraft/unmapped/C_zubvmeye;)I
 		ARG 1 pos
+	METHOD m_ewsshoyd getFreeChunkTicketsAt (Lnet/minecraft/unmapped/C_hynzadkk;)I
+	METHOD m_fcmsgafb getTypePosInCircleSortedByDistance (Ljava/util/function/Predicate;Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_hynzadkk;ILnet/minecraft/unmapped/C_uegwgivt$C_dzezdmdw;)Ljava/util/stream/Stream;
```

Comment:
```suggestion
	METHOD m_fcmsgafb getTypePosInCircleDistanceSorted (Ljava/util/function/Predicate;Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_hynzadkk;ILnet/minecraft/unmapped/C_uegwgivt$C_dzezdmdw;)Ljava/util/stream/Stream;
```
maybe? this name feels less moj

Reply:
`getTypesAndPositionsSorted` (and `getTypesAndPositions`)?